src/ts/signers.ts

```diff
@@ -56,7 +59,7 @@ export class TypedDataV3Signer implements TypedDataSigner {
     const address = await this.getAddress();
 
     // Actual signing
-    return (await this.provider.send("eth_signTypedData_v3", [
+    return (await this.provider.send(this._signMethod, [
```

Comment:
you dont need to return `await` just return it

Reply:
true, although not part of the scope of this PR. Happy to change it if this is really bothering